src/Sulu/Bundle/SecurityBundle/Resources/js/views/Form/toolbarActions/EnableUserToolbarAction.js

```diff
@@ -3,20 +3,21 @@ import {action, observable} from 'mobx';
 import {AbstractFormToolbarAction} from 'sulu-admin-bundle/views';
 import {ResourceRequester} from 'sulu-admin-bundle/services';
 import {translate} from 'sulu-admin-bundle/utils';
-import type {ButtonItemConfig} from 'sulu-admin-bundle/containers/Toolbar/types';
 
 export default class EnableUserToolbarAction extends AbstractFormToolbarAction {
     @observable loading: boolean = false;
 
-    getToolbarItemConfig(): ButtonItemConfig {
-        const userIsEnabled = this.resourceFormStore.data.enabled;
+    getToolbarItemConfig() {
+        if (this.resourceFormStore.data.enabled) {
```

Comment:
I think we also shouldn't show this button when the user is about to be created, right? Usually users created via this form are automatically enabled, and even if they weren't the button does not work.